content/en/docs/chart_template_guide/getting_started.md

```diff
@@ -117,6 +117,8 @@ NAME: full-coral
 LAST DEPLOYED: Tue Nov  1 17:36:01 2016
 NAMESPACE: default
 STATUS: DEPLOYED
+REVISION: 1
+TEST SUITE: None
 
 RESOURCES:
```

Comment:
Ok I will remove this part also. I though it was only there to display what happends on the cluster and was not related to helm output. Will do this tonight. Thx for the feedback

Reply:
Great, thanks